### ISSUE TITLE
NSW DS Pagination prev/next missing disabled class

### ISSUE THREAD

**Issue by eschweitzer78** [eschweitzer78/gps-design-systems-lwc#247]
### Summary

To conform to v3.5.3, the disabled css class should be added to prev/next when aria-disabled is set to true.

### Salesforce Org Type

Production Org

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
